enterprise_access/apps/api/v1/tests/test_views.py

```diff
@@ -1604,14 +1604,26 @@ def test_list_all_redeemable_only_policies(self):
         response = self.client.get(SUBSIDY_ACCESS_POLICY_LIST_ENDPOINT, query_params)
         response_json = self.load_json(response.content)
 
-        # Verify that api response only includes the redeemable policies
-        assert response_json == [
-            dict(SubsidyAccessPolicyRedeemableSerializer([self.redeemable_policy], many=True).data[0])
```

Comment:
IMO, testing a serializer with the same serializer is self-fulfilling.  It's what led to this bug creeping in after all!